src/main/java/com/finantialcontrol/data/db/jpa/entities/UserData.java

```diff
@@ -0,0 +1,58 @@
+package com.finantialcontrol.data.db.jpa.entities;
+
+import com.finantialcontrol.core.domain.Identity;
+import com.finantialcontrol.core.domain.User;
+import com.finantialcontrol.core.domain.UserStatus;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Table(name = "users")
+@Entity
+public class UserData{
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+  private String firstName;
+  private String lastName;
+  private String username;
+  private String email;
+  private String password;
+  private Boolean enabled = true;
```

Comment:
It is confusing and duplicated, you have another class, if I am not mistaken, called SignUpRequest that has the same fields and the same purpose as this?